.github/workflows/build.yml

```diff
@@ -4,10 +4,22 @@ name: Build
 # All jobs will pass without running if no *{.go, .mod, .sum} files have been modified
 on:
   pull_request:
+    paths:
+      - "**.go"
+      - go.mod
+      - go.sum
+      - Makefile
+      - "!test/**"
   merge_group:
   push:
     branches:
       - main
```

Comment:
I know this is not part of this PR but, shouldn't we be running this on every release branch as well?

Reply:
We do run the `build` workflow for every release branch (e.g., [v0.38.x](https://github.com/cometbft/cometbft/blob/v0.38.x/.github/workflows/build.yml#L9)